lib/jekyll/filters.rb

```diff
@@ -363,5 +384,20 @@ def as_liquid(item)
         end
       end
     end
+
+    # Parse a string to a Liquid Condition
+    def parse_condition(exp)
+      parser = Liquid::Parser.new(exp)
+      a = parser.expression
+      if op = parser.consume?(:comparison)
+        b = parser.expression
+        condition = Liquid::Condition.new(a, op, b)
```

Comment:
Our Rubocop setup is setup to accept any style as long as it's not multiple defines.  So if you like to indent on the var like:

```ruby
var = \
  if blah
    var
  else
    var
  end
```

It'll accept it.  Those two are just recommendations, the former can be added.